scripts/build

```diff
@@ -53,9 +53,17 @@ if [[ ${LOCAL_MONGODB:-} == "true" ]] ; then
 fi
 
 if [[ "${JOB_NUM##*.}" == "1" ]]; then
+  # Compiling `connector` can take awhile, so allow it some extra time when
+  # running on travis to avoid having the build killed due to lack of output.
+  if [[ ${TRAVIS:-} ]]; then
+    travis_wait "$SBT" -DisIsolatedEnv=${ISOLATED_ENV:=false} clean coverage connector/test:compile
```

Comment:
The default `travis_wait` is 20, right? But our build usually fails at ~30 minutes, so we probably need to use something higher than that.

`travis_wait` doesn’t work as documented. What it _actually_ does is run for `x` minutes without the no-output timeout, then forcibly quits the program after `x` minutes.